.pre-commit-config.yaml

```diff
@@ -0,0 +1,49 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.1.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-docstring-first
+    -   id: double-quote-string-fixer
+    -   id: check-ast
+    -   id: check-yaml
+    -   id: debug-statements
+    -   id: name-tests-test
+    -   id: requirements-txt-fixer
```

Comment:
I recommend you to move all hooks which modify files to the top of the list. So that checks which just validate files would consistently consume files after transformation.